### PR TITLE
fix(misc): show full length of agenda item text when hovering

### PR DESCRIPTION
**Pull request by tianrunhe**
## Testing scenarios
- [ ] Agenda Item at team dash
  - Hover an agenda item at team dash page
  - See full length of the text in the tooltip
  - Delete/Pin/Unpin still works

- [ ] Agenda Item in check-in meeting
  - Hover an agenda item in a check-in meeting
  - See full length of the text in the tooltip
  - Delete/Pin/Unpin still works

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label https://github.com/ParabolInc/parabol/labels/Skip%20Maintainer%20Review if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/MeetingSubnavItem.tsx [hunk 1]
```diff
@@ -98,7 +111,18 @@ const MeetingSubnavItem = (props: Props) => {
       isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
       onClick={!isDisabled ? onClick : undefined}
     >
-      <ItemLabel isComplete={isComplete}>{children}</ItemLabel>
+      <Tooltip open={openTooltip}>
+        <TooltipTrigger asChild>
+          <ItemLabel
+            isComplete={isComplete}
+            onMouseEnter={handleMouseEnter}
```
Comment (nickoferrall) [inline packages/client/components/MeetingSubnavItem.tsx:118]: -1 the radix tooltip automatically shows the tooltip when you hover over the item, so adding `onMouseEnter` will conflict with this behaviour

File: packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx [hunk 2]
```diff
@@ -238,7 +238,9 @@ const AgendaItem = (props: Props) => {
         />
       </AgendaItemStyles>
       {tooltipPortal(
-        pinned ? `Unpin "${content}" from every check-in` : `Pin "${content}" to every check-in`
+        pinned
+          ? `Unpin this agenda topic from every check-in`
```
Comment (nickoferrall) [inline packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx:242]: +1 why remove the "content" bit out of interest? The tooltip updates the position so we can always see it. I prefer being able to see the content of the tooltip I'm pinning so I'm sure I'm pinning the right one

**Review by nickoferrall — CHANGES_REQUESTED** [review]
You assigned me to the PR but I assume you meant reviewer, so here is my review 😄 

Ideally, I think we should only show the tooltip when you can't see the entire agenda item. Most of the time, you can read the full agenda item, so you wouldn't want to see a tooltip: https://www.loom.com/share/498faf0911d24118a8fab147678cd953

File: packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx (hunk 2 above)
Comment (tianrunhe) [reply packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx:242]: I find it redundant & too verbose? I'm trying to follow a "industry standard" (if that's a thing😂). For example, in Arc tab management:

Screenshot 2024-09-18 at 10 39 04

Notice here it says "archive this tab", not "archive 'GitHub'"

**Comment by tianrunhe** [comment]
> You assigned me to the PR but I assume you meant reviewer, so here is my review 😄

Sorry for randomly assigning work to you, this is what happened when you go on a vacation and back 🤣!
Thanks for reviewing this!

**Review by nickoferrall — APPROVED** [review]
LGTM!

**Pull request merged**